zanata-war/src/main/java/org/zanata/ZanataInit.java

```diff
@@ -430,7 +431,8 @@ private void logBanner(VersionInfo ver, AppServerVersion appServerVersion) {
         }
         log.info("  AS version: " + appServerVersion.asVersion.orNull());
         log.info("  SCM: " + ver.getScmDescribe());
-        log.info("  Red Hat Inc 2008-2014");
+        log.info("  Red Hat Inc 2008-{}",
```

Comment:
Just because the code is running in the year X doesn't mean the copyright year of publication has changed.  We should just put the year of publication: 2008.

Wait, is this even a copyright statement?  It doesn't say copyright or ©, so what is it supposed to be?